internal/cmd/egctl/translate.go

```diff
@@ -280,7 +281,7 @@ func translateGatewayAPIToIR(resources *gatewayapi.Resources) (*gatewayapi.Trans
 		GatewayControllerName:   string(resources.GatewayClass.Spec.ControllerName),
 		GatewayClassName:        gwapiv1.ObjectName(resources.GatewayClass.Name),
 		GlobalRateLimitEnabled:  true,
-		EndpointRoutingDisabled: true,
+		EndpointRoutingDisabled: isEndpointRoutingDisabled(resources),
```

Comment:
lets keep this to `true` , if you look at the newly generated yaml we are loosing out on endpoint o/p because the input is not providing related endpointslices

Reply:
so I assume you mean that I have inadvertently flipped the default in `isEndpointRoutingDisabled`, i.e. the default should be `ServiceRoutingType`

Reply:
the default for the system, in the gateway-api layer is `EndpointType`, but for this test case in `cmd/egctl` we've flipped it to `Service` , yah its confusing, sorry :)

Reply:
I have restored the line to read `EndpointRoutingDisabled: true,`